### ISSUE TITLE
Unable to Monitor/Favorite Tags Which Recieve Regular Changes

### ISSUE THREAD

**Issue by SudosCode** [Bionus/imgbrd-grabber#2924]
Tags which receive regular updates, but not from new uploaded images, cannot be properly monitored. For instance, I would like to be able to monitor the tag yande.re uses to categorize a user's favorites (vote:>=3:Username), however when I attempt to do so, the monitor doesn't recognize any new images which were added to this tag. My understanding is that this is because new images are archived as 'checked' when the monitor is ticked, and is then ignored and never checked again if it falls into the monitored tag. This only allows for images which are both uploaded and added to the tag within each monitor tick to be downloaded, and no monitor interval or delay will be able to work for this.

**Comment by Bionus** [comment]
Yes, monitors (and favorites, for that matter) work by checking all results of a given search query that were uploaded since the last check / monitor run. The delay can allow some leeway to give time for uploaders to tag their images, but if "old" images are added to the results, that won't work.

What solution do you suggest for this? Given the goal is to __not__ have to go through every single page of results when running monitors, and trying to only have 1 HTTP query per monitor run.

**Comment by SudosCode** [comment]
While I don't know how the API for the imageboards themselves work, I assume that given that Grabber can remember which images have been checked already it could be possible to cross check the images which have already been checked for a given tag against the images that are present in a request to the imageboard. This would solve the problem for tags which always get the new images added to the front (which in my same example can be done via order:vote), but if an image would be added somewhere further into the list (like in a pool) then this wouldn't work without checking every page. Allowing to specify how many images deep to cross check against could potentially help (only check the newest image against the newest stored image, etc.).
I don't think a way without checking every page exists, but this kind of tag is pretty niche and it should be able to cover pretty much every practical use case.
